tailwind.config.ts

```diff
@@ -69,6 +97,9 @@ const config = {
           to: { height: "0" },
         },
       },
+      screens: {
+        xs: "420px",
```

Comment:
```suggestion
        xs: "430px",
```
The mobile frame is 430, but we likely will want to use higher breakpoints anyway (i.e. shift all the sizes and formatting to mobile version well before 430px) so being exact here shouldn't matter.